### PR TITLE
2613 link

### PR DESCRIPTION
**Pull request by jdpagley**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@shoflo/react-draft-wysiwyg",
-  "version": "1.6.0",
+  "version": "1.7.0",
```
Comment (MandarinConLaBarba) [inline package.json:3]: Did the API change? Don't think we added any new properties really

Comment (MandarinConLaBarba) [reply package.json:3]: Thinking this should just be a patch, what do you think?

Comment (jdpagley) [reply package.json:3]: I normally bump minor if there are new additions. It is a really small piece of added functionality so I could bump patch. When do you bump minor vs patch?

Comment (MandarinConLaBarba) [reply package.json:3]: Actually I think it's fine...

**Pull request merged**